Documentation/project-docs/glossary.md

```diff
@@ -105,41 +242,94 @@ var odds = source.Where(obj => obj.Id == 1).ToArray();
+to a SQL WHERE clause: `WHERE ID = 1`, and then executed on the SQL server.
 
 #### Parallel LINQ
 
 **Also referred to as**: PLINQ
 
-Also introduced in .NET Framework 3.5 Parallel LINQ. Parallel LINQ has a subset of the methods the LINQ does but may execute the iterations on different threads in any order. Generally to use Parallel LINQ you would just call the `AsParallel()` method on a collection implementing `IEnumerable`. And if at any point you wanted to return to "normal LINQ you can just call `AsSequential()`.
+Also introduced in .NET Framework 3.5 Parallel LINQ. Parallel LINQ has a subset
+of the methods the LINQ does but may execute the iterations on different threads
+in any order. Generally to use Parallel LINQ you would just call the
+`AsParallel()` method on a collection implementing `IEnumerable`. And if at any
+point you wanted to return to "normal LINQ you can just call `AsSequential()`.
 
 ### JSON.NET
 
-Released in June 2006 by [James Newton-King](https://twitter.com/JamesNK), JSON.NET has become the defacto standard for JSON serialization and deserialization in .NET. It is [open source](https://github.com/JamesNK/Newtonsoft.Json) and support almost every platform .NET code can run on (.NET Framework 2.0, 3.0, 3.5, 4.0, and 4.5; Mono; MonoTouch/Xamarin.iOS; MonoDroid/Xamarin.Android; Silverlight 3, 4, and 5; Windows Phone 8, 8, and 8.1; Windows 8 Store; .NET Core).  
+Released in June 2006 by [James Newton-King][JamesNK], JSON.NET has become the
+defacto standard for JSON serialization and deserialization in .NET. It is [open
+source][Newtonsoft.Json] and support almost every platform .NET code can run on
+(.NET Framework 2.0, 3.0, 3.5, 4.0, and 4.5; Mono; MonoTouch/Xamarin.iOS;
+MonoDroid/Xamarin.Android; Silverlight 3, 4, and 5; Windows Phone 8, 8, and 8.1;
+Windows 8 Store; .NET Core).  
 
 ### Windows Forms
 
 **Also referred to as**: WinForms
 
-Windows Forms is an API provided by the .NET Framework (mostly in the `System.Windows.Forms` namespace) for creating desktop applications. Windows Forms provides an event-driven model for application development on top of the native loop-driven Win32 model. Mono [has an implementation](http://www.mono-project.com/docs/gui/winforms/) of Windows Forms, though it is not complete, since some parts of Windows Forms are tied to the Windows platform.
+Windows Forms is an API provided by the .NET Framework (mostly in the
+`System.Windows.Forms` namespace) for creating desktop applications. Windows
+Forms provides an event-driven model for application development on top of the
+native loop-driven Win32 model. Mono [has an implementation][mono-winforms] of
+Windows Forms, though it is not complete, since some parts of Windows Forms are
+tied to the Windows platform.
 
-Windows Forms is in maintenance mode now. That means new features will generally not be added.
+Windows Forms is in maintenance mode now. That means new features will generally
```

Comment:
Is this stated anywhere? I don't speak for that repo, but I don't see it in their [roadmap](https://github.com/dotnet/winforms/blob/master/roadmap.md). Was it in a blog post?

Reply:
Ah, you didn't add it.

Reply:
I did write it back in 2016 when it was official. Now I'm not sure what the status is so I've left that alone so far.